### PR TITLE
Added 'max_insanity'

### PR DESCRIPTION
**Pull request by Elyriand21**

### CODE REVIEW

File: DeveloperConsole.cs [hunk 1]
```diff
@@ -170,8 +203,45 @@ private void CommandRunner(string command)
...
+                            // Returns the player's level
+                            case "level":
+                                result = $"Player {args[1]}'s Level: {LC_Lib.GetLevelNumber(Player)}";
+                                break;
+                        }
+                    }
+                    break;
+
+                // heal_player <player> <value>
+                case "heal_player":
```
Comment (Lillious) [inline DeveloperConsole.cs]: This is redundant as set health already exists, please remove.

**Review by Lillious — CHANGES_REQUESTED** [review]

File: DeveloperConsole.cs [hunk 2]
```diff
@@ -43,7 +44,8 @@ private void CommandRunner(string command)
                             "help - Displays this message\n" +
                             "clear - Clears the command history\n" +
                             "help set - Shows available commands for 'set'\n" +
-                            "help get - Shows available commands for 'get'";
+                            "help get - Shows available commands for 'get'\n" +
+                            "heal_player <player number> <value> - Heals player by <value>";
```
Comment (Lillious) [inline DeveloperConsole.cs]: Please remove as set health exists already

File: DeveloperConsole.cs [hunk 3]
```diff
@@ -7,8 +7,9 @@
 using UnityEngine;
 using UnityEngine.UI;
 using UnityEngine.InputSystem;
+using UnityEngine.InputSystem.LowLevel;
```
Comment (Lillious) [inline DeveloperConsole.cs]: What is this used for?

File: DeveloperConsole.cs (hunk 1 above)
Comment (Elyriand21) [reply DeveloperConsole.cs]: It's funny that as soon as I saw this, I realized the exact same thing

**Review by Lillious — CHANGES_REQUESTED** [review]

File: DeveloperConsole.cs (hunk 3 above)
Comment (Elyriand21) [reply DeveloperConsole.cs]: Okay, I thought it was added in on your side. I'll remove that

**Review by Elyriand21 — COMMENTED** [review]
Successfully removed 'heal_player' from command list

**Review by Lillious — APPROVED** [review]
Looks good

**Pull request merged**